simulation/executor/config.go

```diff
@@ -0,0 +1,83 @@
+package simulation
```

Comment:
I'd rec, lets move the other copied functions into this file, and call it `legacyconfig.go`, as a note to ourselves to come back and better abstract later